fem/pgridfunc.cpp

```diff
@@ -487,16 +487,17 @@ void ParGridFunction::ProjectBdrCoefficientTangent(VectorCoefficient &vcoeff,
 
 void ParGridFunction::Save(std::ostream &out) const
 {
+   double *data_  = const_cast<double*>(ReadAccess(false));
```

Comment:
We only temporarily change the signs of some entries -- this is not new in this PR.

I had to add this line because we are breaking the const-ness of the `Memory` object. Before we were still modifying (temporarily) the contents of `data` and that was not an issue (even in `const` methods, we could modify the contents of `data` but not the pointer `data` itself).